fitbenchmarking/results_processing/tests/test_support_page.py

```diff
@@ -103,10 +103,15 @@ def test_file_name(self):
                                        support_pages_dir=self.dir.name,
                                        options=self.options)
         file_names = [r.support_page_link for r in self.results]
-        expected = [os.path.join(self.dir.name, f)
+        print("*** file_names ***")
```

Comment:
Unless the printing is needed I suggest this is removed

Reply:
Good catch -- fixed